docs/dom-testing-library/api-debugging.mdx

```diff
@@ -40,6 +40,20 @@ This works on macOS/Linux, you'll need to do something else for Windows. If
 you'd like a solution that works for both, see
 [`cross-env`](https://www.npmjs.com/package/cross-env).
 
+**Note**: The output of the DOM is colorized by default if your tests are
+running in a node environment. However, you may sometimes want to turn off
+colors, such as in cases where the output is written to a log file for
+debugging purposes. You can use the environment variable `COLORS` to explicitly
+force the colorization off or on. For example:
+
+```
+COLORS=false npm test
+```
+
+As with `DEBUG_PRINT_LIMIT`, `COLORS` works in macOS/Linux but you'll need to do
+something else for Windows. If you'd like a solution that works for both, see
+[`cross-env`](https://www.npmjs.com/package/cross-env).
```

Comment:
Sorry for the additional comment.
I think we can just copy paste this note, what do you think?

```suggestion
This works on macOS/Linux, you'll need to do something else for Windows. If
you'd like a solution that works for both, see
[`cross-env`](https://www.npmjs.com/package/cross-env).
```

Reply:
Yeah, sure.